solver_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'minitest/autorun'
+require_relative 'solver'
+
```

Comment:
Hi @IvonneBenitesRodriguez, @gilberthappi,

Great effort on the TDD project! 👏 I noticed that the commit history sequence suggests that the Solver class methods were written before being tested. In a Test-Driven Development (TDD) workflow, the typical sequence is to write tests first and then implement the code to make those tests pass. Given the importance of Test-Driven Development (TDD) in this project, I recommend starting fresh with a new repository to implement TDD properly. This will allow you to maintain a clean and accurate commit history aligned with the TDD methodology.

Here's a suggested approach:

1. **Create a New Repository:**
   - Start a new repository specifically for this project.

2. **Initial Commit for Project Setup:**
   - Make an initial commit for setting up the project structure, including any necessary files or configurations.

3. **Follow TDD Workflow:**
   - Begin by writing tests for the Solver class methods before implementing the actual code. Follow the TDD cycle of writing tests, implementing code, and ensuring tests pass.

4. **Commit Each Step:**
   - Make individual commits for each step of the TDD process. This provides a clear and sequential history of your development.

5. **Commit Messages:**
   - Use descriptive commit messages that indicate whether the commit is for writing tests or implementing code. This helps in understanding the purpose of each commit.

Starting fresh will allow you to fully embrace TDD principles and create a structured development history. If you encounter any challenges or have questions during this process, feel free to ask for assistance.

Best of luck with the project! 🌟

Warm regards,
Temitope